pkgs/games/steam/fhsenv.nix

```diff
@@ -47,7 +47,10 @@ let
   ldPath = lib.optionals stdenv.is64bit [ "/lib64" ]
   ++ [ "/lib32" ]
   ++ map (x: "/steamrt/${steam-runtime-wrapped.arch}/" + x) steam-runtime-wrapped.libs
-  ++ lib.optionals (steam-runtime-wrapped-i686 != null) (map (x: "/steamrt/${steam-runtime-wrapped-i686.arch}/" + x) steam-runtime-wrapped-i686.libs);
+  ++ lib.optionals (steam-runtime-wrapped-i686 != null) (map (x: "/steamrt/${steam-runtime-wrapped-i686.arch}/" + x) steam-runtime-wrapped-i686.libs)
+  # allows for steam play and other valve utilities to work
+  ++ lib.optionals stdenv.is64bit [ "\${HOME}/.steam/steam/ubuntu12_64" ]
+  ++ [ "\${HOME}/.steam/steam/ubuntu12_32" ];
```

Comment:
```suggestion
  ++ lib.optionals stdenv.is64bit [ "$HOME/.steam/steam/ubuntu12_64" ]
  ++ [ "$HOME/.steam/steam/ubuntu12_32" ];
```

Reply:
I had this originally, but whitespace will break the path